### PR TITLE
add an optional field for tabulate to support eliminate space line between each data row

### PR DESCRIPTION
**Pull request by zvier**
related to #14 

Signed-off-by: zvier <zvier20@gmail.com>

### CODE REVIEW

**Comment by zvier** [comment]
@bndr please take a look ~

**Comment by jingxiaolu** [comment]
Hi @zvier I think a method `SetDenseMode()` is also needed at tabulate.go?

File: tabulate.go [hunk 1]
```diff
@@ -84,6 +84,7 @@ type Tabulate struct {
 	WrapStrings   bool
 	WrapDelimiter rune
 	SplitConcat   string
+	DenseMode     string
```
Comment (jingxiaolu) [inline tabulate.go:87]: I would prefer bool

**Comment by zvier** [comment]
> Hi @zvier I think a method `SetDenseMode()` is also needed at tabulate.go?

Good idea. PTAL

**Pull request merged**